lib/flatpak-remote.c

```diff
@@ -823,17 +823,16 @@ flatpak_remote_commit (FlatpakRemote   *self,
         }
       else
         {
+          g_autoptr(GError) local_error = NULL;
           gboolean gpg_verify_value;
 
           g_key_file_remove_key (config, group, "collection-id", NULL);
 
           /* Without a collection ID gpg-verify-summary should go back to
            * matching gpg-verify. */
-          gpg_verify_value = g_key_file_get_boolean (config, group, "gpg-verify", error);
-          if (*error == NULL)
+          gpg_verify_value = g_key_file_get_boolean (config, group, "gpg-verify", &local_error);
+          if (local_error == NULL)
             g_key_file_set_boolean (config, group, "gpg-verify-summary", gpg_verify_value);
```

Comment:
If there *is* an error in getting the `gpg-verify` key, should we fail-safe and set `gpg-verify-summary` to `true`? What’s its default value in new configs?

Reply:
OK. This looks good to me then.